### PR TITLE
A single typo fix

### PR DESCRIPTION
**Pull request by tdpeuter**
Fixed recived to be recieved

That's all. There is similar stuff on the wiki, but I cannot edit that.

### CODE REVIEW

**Comment by Toqozz** [comment]
I believe it should actually be received* ;)

**Comment by tdpeuter** [comment]
This is embarrassing, fixed it!

**Comment by Toqozz** [comment]
Neat, thanks!  Would you be able to point out any of the typos on the wiki so I can fix them?

**Comment by tdpeuter** [comment]
The one most thing that I noticed was that I believe `Hook(parent_anchor: XX, self_anchor: XX)` must have been `Hook(parent_hook: XX, self_hook: XX)` previously, because that is inconsistent all over the [Blocks info page](https://github.com/Toqozz/wired-notify/wiki/Blocks). Specifically: 

- `### Optional` > first code block (3 lines) (lines :30, :39, :51 in file)
- `### Optional` > In explanation text (4 times) (line :70 in file)
- `### Real example` > first code block (in every block, one line) (lines :83, :95, :107, :121 in file)
- `### Render criteria` > Code block (1 time) (line :169 in file)

Basically every `Hook(...)` format in the page, except for the first one that has been edited or written afterwards if my guess is correct. If I do the 'translation' myself, I get something like [this](https://gist.github.com/tdpeuter/9849e6aa16ec12c5a9322e376f743809). 

If this was renamed and if Hook is a self-defined property (I couldn't immediately find one in the code though), is it an idea to ditch the Hook keyword altogether and go for Anchor or something?

**Comment by Toqozz** [comment]
Should be fixed now!  Yeah, I think `anchor` used to be `hook`.
